### PR TITLE
Update dependency versions

### PR DESCRIPTION
**Pull request by SasinduDilshara**
$Update dependency versions

### CODE REVIEW

**Comment by MadhukaHarith92** [comment]
Since some dependencies got bumped into the next minor version, let's bump the transaction module to the next minor version.

**Comment by SasinduDilshara** [comment]
> Since some dependencies got bumped into the next minor version, let's bump the transaction module to the next minor version.

will update

File: gradle.properties [hunk 1]
```diff
@@ -4,22 +4,22 @@ group=org.ballerinalang
 version=1.5.1-SNAPSHOT
```
Comment (kalaiyarasiganeshalingam) [inline gradle.properties:4]:
```suggestion
version=1.6.0-SNAPSHOT
```

File: gradle.properties [hunk 2]
```diff
@@ -4,22 +4,22 @@ group=org.ballerinalang
...
 
 ballerinaLangVersion=2201.5.0
```
Comment (kalaiyarasiganeshalingam) [inline gradle.properties:6]:
```suggestion
ballerinaLangVersion=2201.6.0-20230519-135600-d887142d
```

Comment (SasinduDilshara) [reply gradle.properties]: Updated

File: gradle.properties (hunk 1 above)
Comment (SasinduDilshara) [reply gradle.properties]: Updated

File: gradle.properties (hunk 2 above)
Comment (MadhukaHarith92) [reply gradle.properties]: @kalaiyarasiganeshalingam do we need to update the lang version?

Comment (kalaiyarasiganeshalingam) [reply gradle.properties]: Sorry, we don't need to update to update6. I thought the HTTP lang version was updated to update 6.

File: gradle.properties [hunk 3]
```diff
@@ -1,25 +1,25 @@
 org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
-version=1.5.1-SNAPSHOT
+version=1.6.0-SNAPSHOT
 
-ballerinaLangVersion=2201.5.0
-stdlibIoVersion=1.4.0
-stdlibConstraintVersion=1.1.0
+ballerinaLangVersion=2201.6.0-20230519-135600-d887142d
```
Comment (kalaiyarasiganeshalingam) [inline gradle.properties:6]:
```suggestion
ballerinaLangVersion=2201.5.0
```

**Review by MadhukaHarith92 — APPROVED** [review]

**Pull request merged**